src/ui/public/vislib/lib/dispatch.js

```diff
@@ -310,5 +311,11 @@ export default function DispatchClass(Private) {
   }
 
 
+  function justifyOpacity(opacity) {
+    const decimalNumber = parseFloat(opacity, 10);
+    return (0 <= decimalNumber  && decimalNumber <= 1) ? decimalNumber : 0.5;
```

Comment:
I thought about that too, and I think I prefer the fallback:
- it's easier to explain:
- edge cases don't need additional logic (e.g. users putting in a string we probably should default to a sensible value)

anyway, I don't feel strongly about this, so if you have preference for the clamping, I'm happy to oblige.